pages/500.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const error500: React.FC = () => {
+  return (
+    <div className="h-screen p-16 flex flex-col px-5 mx-auto justify-center items-center">
+      <h1 className="mb-10 font-extrabold text-8xl">500</h1>
+      <p className="font-semibold text-2xl">Uh oh. Something broke on our end. Check back later.</p>
+    </div>
+  );
+};
+
+export default error500
```

Comment:
What you did works here too but just to keep everything more consistent within our project, we're going to stick with exporting with the function declaration. So remove line 12 and just replace your function declaration (line 3) with `export default function error500() { ... }`